week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,9 +35,13 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
-}
+  
+  let removed=arr.spLice(3,1);
+
+      return removed; // complete this statement
 
+    }
+      
```

Comment:
Again code formatting :)
Here is a resource about JavaScript Style Guide and Conventions if you would like to see some examples:
 * https://www.w3schools.com/js/js_conventions.asp